data/widgets/options.xml

```diff
@@ -301,10 +301,12 @@
 
           <vbox>
             <check id="undo_goto_modified"
-                   text="@.undo_goto_modified"
-                   tooltip="@.undo_goto_modified_tooltip" />
+                  text="@.undo_goto_modified"
+                  tooltip="@.undo_goto_modified_tooltip" />

```

Comment:
Did something changed in these lines?(I think only whitespace was modified here)